reference/api/keys.md

```diff
@@ -218,9 +275,9 @@ Date and time when the key will expire, represented in RFC 3339 format. `null` i
 
 ## Update a key
 
-<RouteHighlighter method="PATCH" route="/keys/{key}"/>
+<RouteHighlighter method="PATCH" route="/keys/{uid}"/>
```

Comment:
Good question. There is no particular reason, it has not been thought out. We only imagined it for GET.

It would indeed be interesting to facilitate the user experience. WDYT @Kerollmops is it a change that can be made?

Reply:
Technically speaking it must be possible to be able to update a key by either specifying the UID or the key itself. I don't know why we would allow both to be specified thought.

Reply:
You know what? The core team managed it directly. @curquiza and I just checked it out. I will update the spec! 

It is possible to specify an API Key resource for the PATCH and DELETE using `:uid_or_key` as well on the GET verb.

We can find some happiness in misfortune. 😄

Reply:
Consistency makes me happy. We'll update the docs accordingly.